css/style.css

```diff
@@ -68,23 +71,29 @@ th {
 
 #form-con {
   text-align: center;
+  width: 400px;
+  margin: auto;
 }
 
 #submit-con:hover {
   color: blue;
 }
-
-label {
-  border: 3px blue solid;
+#fname {
+  border: 2px solid red;
+  border-radius: 10px;
 }
-
-.home {
-  height: 100vh;
-  background-image: url(/images/madison-sunset.jpg);
-  background-position: center;
-  background-size: contain;
-  background-repeat: no-repeat;
-  /* background-color: red; */
+#lname {
```

Comment:
Deleting lines 81 to 87 affects `index.html` could you put them in? Do you want me to put them back in and make a commit to your branch?